fullstack/go/price-calculator/cmd/calculator/main.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
Sorry for nitpicking but should we use spaces instead of tabs? 😛

Reply:
The question is not whether we should, the question is whether `gofmt` should've 😆 

From the [documentation](https://pkg.go.dev/cmd/gofmt#section-documentation):

> Gofmt formats Go programs. It uses tabs for indentation and blanks for alignment. Alignment assumes that an editor is using a fixed-width font.

We have zero control over this I'm afraid.

Reply:
Now that makes me wonder how the old code was in spaces, or if I was very, very wrong.

Reply:
It could be that it wasn't run through `gofmt`, but if it wasn't, it was _it_ that was wrong, not you!